vue.config.js

```diff
@@ -26,12 +28,32 @@ module.exports = {
         },
       ],
     },
-    externals: process.env.NODE_ENV === 'production' ? [
-      'vue',
-      /^bootstrap\/.+$/,
-      /^@processmaker\/.+$/,
-      'i18next',
-      '@panter/vue-i18next',
-    ] : [],
+    externals: (() => {
+      const externals = [];
+      if (process.env.NODE_ENV === 'production') {
+        externals.push(
+          'vue',
+          /^bootstrap\/.+$/,
+          /^@processmaker\/.+$/,
+          'i18next',
+          '@panter/vue-i18next',
+          'jointjs',
+          'luxon',
+          'bpmn-moddle',
+          'lodash',
```

Comment:
Many of these dependencies were also being included in the process maker project. Including them in modeler was just causing bloat so I moved to exclude them..